src/dmd/mtype.d

```diff
@@ -1961,6 +1961,38 @@ extern (C++) abstract class Type : RootObject
         return t;
     }
 
+    /************************************
+     * Add MODxxxx bits to type of __mutable member variable
```

Comment:
```
Returns:
    type with added bits
```

Reply:
Redundancy in documentation is sometimes useful, e.g. if the `Returns` information was also mentioned in a *long* description, so the return can be quickly looked up. Here, reading the *short* description is clearly enough IMO.